package/base-files/files/sbin/sysupgrade

```diff
@@ -158,7 +158,7 @@ list_changed_conffiles() {
 list_static_conffiles() {
 	local filter=$1
 
-	find $(sed -ne '/^[[:space:]]*$/d; /^#/d; p' \
+	find $(sed -ne 's/^[[:space:]]*//; /^$/d; /^#/d; p' \
 		/etc/sysupgrade.conf /lib/upgrade/keep.d/* 2>/dev/null) \
 		\( -type f -o -type l \) $filter 2>/dev/null
```

Comment:
(to avoid sed and lengthy regex) The whole expression could be revised to:
```sh
find $(grep -vhE '^\s*(#|$)' /etc/sysupgrade.conf /lib/upgrade/keep.d/* 2>/dev/null) \( -type f -o -type l \) $filter 2>/dev/null
```
I don't see that `$filter` is used, however?